katsdptelstate/aio/redis.py

```diff
@@ -68,6 +80,26 @@ async def __call__(self, client: aioredis.Redis,
         return await client.eval(self.script, keys, args)
 
 
+def _unpack_query_timestamp(packed_timestamp: bytes) -> Tuple[bytes, bool]:
+    """Split the result of :func:`katsdptelstate.utils.pack_query_timestamp`.
+
+    Returns
+    -------
+    value
+        Boundary value
+    include
+        Whether the boundary value is inclusive (false for ``-`` and ``+``).
+    """
+    if packed_timestamp in {b'-', b'+'}:
+        return packed_timestamp, False
+    elif packed_timestamp[:1] == b'[':
```

Comment:
I like `[:1]` instead of `[0]` - handy trick that.

Reply:
Also necessary for byte strings in Python 3 if you want a byte string back: `b'xy'[0]` is `120`.